frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/GitPushDialog.java

```diff
@@ -64,6 +80,30 @@ private void push() {
     push(isForcePushRequired);
   }
 
+  /**
+   * Overridden to provide a way to perform an action once the push is completed successfully.
+   * Compare to the {@link com.intellij.dvcs.push.ui.VcsPushDialog#push(boolean)}
+   * which doesn't implement {@code onSuccess} callback.
+   */
+  @Override
+  @UIEffect
+  public void push(boolean forcePush) {
+
+    String title = DvcsBundle.message("push.process.pushing");
```

Comment:
nit: it's dangerous to rely on their classes and bundles here. they can change DvcsBundle to GitBundle someday, the key, or the value

Reply:
Right... replaced with an identical string from our bundle